app/helpers/external/mailbluster_helper.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'httparty'
+
+module External
+  module MailblusterHelper
+    MAILBLUSTER_URL = 'https://api.mailbluster.com/api/leads/'
+
+    # def query_api(user, method)
```

Comment:
This is generally a good idea. Though for now the code is quite readable. I would probably make the `method` be first, and instead of `user`, `body` probably makes the most sense. 

Actually I think I would do
`(method, path, body)` where the path is either empty or the email hash passed in.